plonk/src/transcript/solidity.rs

```diff
@@ -117,6 +116,11 @@ impl<F: PrimeField> PlonkTranscript<F> for SolidityTranscript {
             b"input size",
             vk.num_inputs.to_be_bytes().as_ref(),
         )?;
+        <Self as PlonkTranscript<F>>::append_message(
+            self,
+            b"EVM word alignment padding",
+            &[0u8; 12],
```

Comment:
Why 12?

Reply:
the first 3 fields are 4 + 8 + 8 = 20 bytes, and word size in EVM is 32 bytes,
I want to align with memory boundary, so that I can write easier assembly later (in my current WIP work)